docs/README.md

```diff
@@ -180,9 +185,13 @@ Avoid, especially with gerunds.
 For example, prefer the "The feature writes to…" over "The feature provides writing to…."
 ([#727](https://github.com/web-platform-dx/web-features/pull/727#discussion_r1537635491))
 
+##### represents
+
+If an HTML element or other type doesn't obviously do anything, then it probably _represents_ something.
```

Comment:
I'd really like to see a Good/Bad example here. I'm still struggling with this one a bunch, and having a couple of examples would help a lot.

Reply:
It's hard to think of bad examples, rather I'd treat "represents" as a default for HTML elements which we diverge from when we think some other verb is clearer.

Reply:
Yes, @foolip captured my meaning here, as using it as a default. That said, I was able to come up with some examples, so I added them. This necessitated some edits to actual descriptions, of course.